src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -105,7 +105,7 @@ JobExecutorType(DistributedPlan *distributedPlan)
 			}
 
 			ereport(DEBUG1, (errmsg(
```

Comment:
should update the comment above as well

Reply:
Oh I saw this after merging, I will create another PR and update this.